backend/src/main/java/app/linkedout/services/UsersService.java

```diff
@@ -0,0 +1,53 @@
+package app.linkedout.services;
+
+import app.linkedout.models.Users;
+import app.linkedout.repositories.UsersRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class UsersService implements CrudService<Users, Long> {
+
+    private final UsersRepository usersRepository;
+
+    public UsersService(UsersRepository usersRepository) {
+        this.usersRepository = usersRepository;
+    }
+
+    @Override
+    public List findAll() {
+        ArrayList<Users> ls = new ArrayList<>();
+        usersRepository.findAll().forEach(ls::add);
+        return ls;
```

Comment:
I don't really know spring that well but doesn't findAll() return a List? Can't we simply put return `usersRepository.findAll();`?

Reply:
Doesn't work nvm.